src/bidiMapper/modules/context/BrowsingContextImpl.ts

```diff
@@ -575,33 +578,39 @@ export class BrowsingContextImpl {
 
     this.#cdpTarget.cdpClient.on('Page.javascriptDialogClosed', (params) => {
       const accepted = params.result;
-
+      assert(
```

Comment:
Just return here else this will be `uncaugh` .

Reply:
good point. Added log and return `unknown` to prevent clients from hanging waiting for the event